### PR TITLE
TIP-447: Fix unknown scenarii

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: features/Context/Page/Attribute/Creation.php [hunk 1]
```diff
@@ -53,12 +53,16 @@ public function findField($name)
...
     public function addOption($name, array $labels = [])
     {
-        if (!$this->getElement('attribute_option_table')->find('css', '.attribute_option_code')) {
+        if (null === $this->getElement('attribute_option_table')->find('css', '.attribute_option_code')) {
             $this->getElement('add_option_button')->click();
-            $this->getSession()->wait($this->getTimeout());
+
+            $this->spin(function () {
+                return $this->getElement('attribute_option_table')->find('css', '.attribute_option_code');
+            }, 'The click on new option have not added a new line.');
```
Comment (willy-ahva) [inline features/Context/Page/Attribute/Creation.php]: has

File: features/Context/Page/Attribute/Creation.php [hunk 2]
```diff
@@ -53,12 +53,16 @@ public function findField($name)
      * Add an attribute option
      *
      * @param string $name
+     * @param array  $labels
      */
     public function addOption($name, array $labels = [])
     {
-        if (!$this->getElement('attribute_option_table')->find('css', '.attribute_option_code')) {
+        if (null === $this->getElement('attribute_option_table')->find('css', '.attribute_option_code')) {
```
Comment (juliensnz) [inline features/Context/Page/Attribute/Creation.php]: could you spin directly?

Comment (pierallard) [inline features/Context/Page/Attribute/Creation.php]: @juliensnz Don't like to have "click" on a spin.

**Comment by juliensnz** [comment]
GTM after comment fix

**Comment by damien-carcel** [comment]
GTM

**Pull request merged**